pkgs/by-name/da/daggerfall-unity/package.nix

```diff
@@ -0,0 +1,110 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  fetchzip,
+  lib,
+  libGL,
+  libXScrnSaver,
+  libXcursor,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libudev0-shim,
+  makeDesktopItem,
+  nix-update-script,
+  stdenv,
+  vulkan-loader,
+}:
+let
+  docFiles = [
+    (fetchurl {
+      url = "https://cdn.bethsoft.com/bethsoft.com/manuals/Daggerfall/daggerfall-en.pdf";
+      hash = "sha256-24KSP/E7+KvSRTMDq63NVlVWTFZnQj1yya8wc36yrC0=";
+      meta.license = lib.licenses.unfree;
+    })
+    (fetchurl {
+      url = "https://www.dfworkshop.net/static_files/daggerfallunity/Daggerfall%20Unity%20Manual.pdf";
+      hash = "sha256-FywlD0K5b4vUWzyzANlF9575XTDLivbsym7F+qe0Dm8=";
+      name = "Daggerfall Unity Manual.pdf";
+      meta.license = lib.licenses.mit;
+    })
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "daggerfall-unity";
+  version = "1.1.1";
+
+  src = fetchzip {
+    url = "https://github.com/Interkarma/daggerfall-unity/releases/download/v${finalAttrs.version}/dfu_linux_64bit-v${finalAttrs.version}.zip";
+    hash = "sha256-JuhhVLpREM9e9UtlDttvFUhHWpH7Sh79OEo1OM4ggKA=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    alsa-lib
+    libGL
+    libXScrnSaver
+    libXcursor
+    libXi
+    libXinerama
+    libXrandr
+    libXxf86vm
+    libpulseaudio
+    libudev0-shim
+    vulkan-loader
+  ];
+
+  strictDeps = true;
+
+  installPhase =
+    ''
+      runHook preInstall
+
+      mkdir --parents "$out/share/doc/"
+      cp --recursive * "$out"
+    ''
+    + (lib.strings.concatStringsSep "\n" (
+      map (docFile: ''
+        cp "${docFile}" "$out/share/doc/${docFile.name}"
+      '') docFiles
+    ))
+    + ''
+      runHook postInstall
+    '';
+
+  appendRunpaths = map (pkg: "${lib.getLib pkg}/lib") (finalAttrs.buildInputs);
```

Comment:
Can we use lib.makeLibraryPath here?

Reply:
this might work tho the script expects an array of paths not a colon separated string

Reply:
> this might work tho the script expects an array of paths not a colon separated string

Is the current solution what you expected? Because it does work now. If not, please let me know what you'd want it to look like, as in that case I'm not certain what you mean.